test/integration/task_runner_test.dart

```diff
@@ -49,6 +53,9 @@ Future<TasksRun> runTasks(String projectPath) async {
     if (line.contains(successfulTesting)) {
       successfulTasks.add(successfulTesting);
     }
+    if (line.contains(failedTesting)) {
+      failedTasks.add(failedTesting);
```

Comment:
Would it be possible to output which task failed? All I'm seeing at the end of running the task-runner is something like this:

![image](https://cloud.githubusercontent.com/assets/8091340/25762207/1209b632-319b-11e7-8b19-f33bfabd5675.png)

Then I have to scroll through the output to find what actually failed. Unfortunately at least locally finding the error isn't necessary easy because of the formatting it just blends in with output from the other tasks:

![image](https://cloud.githubusercontent.com/assets/8091340/25762250/46f6bd9a-319b-11e7-8a0d-436c15d74231.png)